staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness_test.go

```diff
@@ -186,6 +186,7 @@ func newApfHandlerWithFilter(t *testing.T, flowControlFilter utilflowcontrol.Int
 
 func TestApfSkipLongRunningRequest(t *testing.T) {
 	epmetrics.Register()
+	fcmetrics.Register()
```

Comment:
Same here - can we instead register them in newApf* helpers?

Reply:
[Maybe also move this cleanup (together with the maxinflight_test.go file) to a separate cleanup PR.]

Reply:
#110516